hack/generator/pkg/codegen/pipeline_simplify_definitions.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package codegen
+
+import (
+	"context"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/astmodel"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/klog"
+)
+
+// simplifyDefinitions creates a pipeline stage that removes any wrapper types prior to actual code generation
+func simplifyDefinitions() PipelineStage {
+	return MakePipelineStage(
+		"simplifyDefinitions",
+		"Simplify definitions",
```

Comment:
Maybe make this textual description a bit closer to the comment above (about removing wrapper types and whatnot)?